pkg/reconciler/common/releases.go

```diff
@@ -132,9 +136,49 @@ func componentDir(instance v1alpha1.KComponent) string {
 	return ""
 }
 
+func componentURL(version string, instance v1alpha1.KComponent) string {
+	manifests := instance.GetSpec().GetManifests()
+	// Create the comma-separated string as the URL to retrieve the manifest
+	if len(manifests) == 0 {
+		return ""
+	}
+
+	urls := make([]string, 0, len(manifests))
+	for _, manifest := range manifests {
+		url := strings.ReplaceAll(manifest.Url, versionVariable, version)
```

Comment:
Would using [`os.Expand`](https://golang.org/pkg/os/#Expand) with the following function be easier to extend in the future:

```
func varExpander(vars map[string]string) func(string) string {
  return func(s string) {
    return vars[s]
  }
}
```

(Obviously, this could be a later refactor, so it's not a priority.)

Reply:
Yep, we can change it later.